regression/ebmc/smv/smv3.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE broken-smt-backend
```

Comment:
Was this always broken, and we just didn't notice? At least it's not clear to me how this PR would newly break the test.

Reply:
It was broken before; we didn't notice since k-induction was hard-wired to use SAT/boolbvt.